skimage/measure/_regionprops.py

```diff
@@ -631,7 +633,7 @@ def _parse_docs():
     import re
     import textwrap
 
-    doc = regionprops.__doc__
+    doc = _regionprops_doc
```

Comment:
`doc = regionsprops.__doc__ if regionprops.__doc__ else ""`

Reply:
this makes test_regionprops to fail when run with `python -OO -mpytest`

Reply:
Really you should just `return` early.

Those tests are likely failing because they expect the docstrings to exits. If you want the tests to pass in `-OO` mode, you will need to change their behaviour in `-OO` mode. Instead of having mangled doc strings, the doc strings should be `None` it seems.